src/locale/nb.js

```diff
@@ -2,54 +2,54 @@
 //! locale : Norwegian Bokmål [nb]
 //! authors : Espen Hovlandsdal : https://github.com/rexxars
 //!           Sigurd Gartmann : https://github.com/sigurdga
+//!           Ronny Hanssen : https://github.com/spiralis
 
 import moment from '../moment';
 
 export default moment.defineLocale('nb', {
-    months : 'januar_februar_mars_april_mai_juni_juli_august_september_oktober_november_desember'.split('_'),
-    monthsShort : 'jan._feb._mars_april_mai_juni_juli_aug._sep._okt._nov._des.'.split('_'),
-    monthsParseExact : true,
-    weekdays : 'søndag_mandag_tirsdag_onsdag_torsdag_fredag_lørdag'.split('_'),
-    weekdaysShort : 'sø._ma._ti._on._to._fr._lø.'.split('_'),
-    weekdaysMin : 'sø_ma_ti_on_to_fr_lø'.split('_'),
-    weekdaysParseExact : true,
-    longDateFormat : {
-        LT : 'HH:mm',
-        LTS : 'HH:mm:ss',
-        L : 'DD.MM.YYYY',
-        LL : 'D. MMMM YYYY',
-        LLL : 'D. MMMM YYYY [kl.] HH:mm',
-        LLLL : 'dddd D. MMMM YYYY [kl.] HH:mm'
+    months: 'januar_februar_mars_april_mai_juni_juli_august_september_oktober_november_desember'.split('_'),
+    monthsShort: 'jan_feb_mar_apr_mai_jun_jul_aug_sep_okt_nov_des'.split('_'),
```

Comment:
According to the language council, the short form of the shortest months is the long form. April is the longest month to write fully in short version. If we know of cases that will break with more than three letters, we should change it. The dot is also part of the shorter form for a lot of abbreviations, but not all (https://www.sprakradet.no/sprakhjelp/Skriveregler/Forkortinger/).

Reply:
I hear the official language council and I'd respect that. But, that is still _really_ weird. IMHO all short forms should have the same length. 

I looked at the link you provided, there it says:

![image](https://user-images.githubusercontent.com/64342/62733051-1b3cbe00-ba26-11e9-868c-5100ef03e01b.png)
Looking at that it seems like the short form is actually `apr.`. Where do they say that it is `april`?

Sidenote: As a programmer it feels weird that they are not just abbreviated to 3 characters for the short form, as a fixed rule.

The typical use of these are not in prosa, it is in tables as headers. And for that the fixed length thing, without dots makes the most sense. So, maybe these are actually different use cases...?

Reply:
Sorry, I misunderstood your comment. My fault :)
April is the shortest *to* change :).